src/components/dls/Spinner/Spinner.module.scss

```diff
@@ -16,6 +16,10 @@
   box-sizing: border-box;
 }
 
+.centered {
```

Comment:
another quick question 😁.
what if we make this styling on the parent component. Is it more complicated in this context? or shall we just make it centered by default?

I don't have strong opinion on this